README.md

```diff
@@ -234,7 +234,10 @@ prettier.format(source, {
   jsxBracketSameLine: false,
 
   // Which parser to use. Valid options are 'flow' and 'babylon'
-  parser: 'babylon'
+  parser: 'babylon',
+
+  // Whether to use a semicolon-free style, similar to eslint's semi: never
+  noSemi: false
```

Comment:
can you name this option `semi` and default to true. The option parser library will make `--no-semi` work